src/cloudWatchLogs/cloudWatchLogsUtils.ts

```diff
@@ -120,4 +120,26 @@ export function findOccurencesOf(document: vscode.TextDocument, keyword: string)
     return ranges
 }
 
+const HIGHLIGHTER = vscode.window.createTextEditorDecorationType({
```

Comment:
```suggestion
const searchHighlight = vscode.window.createTextEditorDecorationType({
```

Reply:
`const` is an interesting keyword is Javascript because all it means is that the variable won't be re-assigned. The original intent of UPPER_CASE naming is to express true universal constants used as configuation that would be in-lined into the executable. The string 'abcde' will never change regardless of execution environment whereas something like `vscode.window.createTextEditorDecorationType` is most definitely not a universal constant.

I think there's at least some value in UPPER_CASE variable names in that they stand out a lot. Still, most things developers normally make UPPER_CASE really shouldn't be. Either because the value is only used in one place or because it's obvious from context what the value represents. Might as well enforce camelCase.